package.json

```diff
@@ -25,31 +25,54 @@
     "name": "The AsyncAPI maintainers"
   },
   "scripts": {
-    "test": "echo 'Nothing To Test'",
+    "start": "npm run build && cross-env NODE_ENV=production node dist/server.js",
+    "dev": "cross-env NODE_ENV=development nodemon",
+    "build": "tsc",
+    "test": "cross-env NODE_ENV=test jest --detectOpenHandles",
```

Comment:
Pretty sure `--detectOpenHandles` should only be added when you run into problems with tests and not as a standard 🤔

Reply:
I have some problems and forgot about that  🦅 👁️  Thanks!